assets/js/app/Instance.js

```diff
@@ -264,19 +262,23 @@ LocusZoom.Instance.prototype.mapTo = function(chr, start, end){
         this.remap_promises.push(this.panels[id].reMap());
     }
 
-    // When all finished update download SVG link
-    Q.all(this.remap_promises).then(function(){
-        this.controls.setBase64SVG();
-    }.bind(this), function(error){
-        console.log(error);
-        this.curtain.drop(error);
-    }.bind(this));
+    Q.all(this.remap_promises)
+        .then(this.mapToComplete)
+        .catch(function(error){
+            console.log(error);
+            this.curtain.drop(error);
+        }.bind(this));
 
     return this;
     
 };
 
+// Logic to fire when an instance has finished mapping to a new region
+LocusZoom.Instance.prototype.mapToComplete = function(){
```

Comment:
I'm having a hard time figuring out what this function is supposed to be for from the name. It's for custom event overriding? Is it part of the public API? Should it be `onMapComplete` for an event?